src: moving f function call comment

danbev opened the pull request:
##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] `make -j4 test` (UNIX), or `vcbuild test nosign` (Windows) passes
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)
src

##### Description of change

The comment about calling the f function seems to have drifted
a little. Moving it to be closer to the actual call.

addaleax commented:
LGTM

danbev commented:
CI: https://ci.nodejs.org/job/node-test-pull-request/3939/

jasnell commented:
LGTM

Trott commented:
The Raspberry Pi failure is unrelated, of course, and due to some ongoing difficulties that have hopefully been resolved now, but in any event, don't need to hold this up IMO.

danbev commented:
Landed in a01e8bcf189bd598d496f347f60007bc2211e528

Fishrock123 commented:
Why was this landed without addressing the comments? I'm 99% certain I posted those here, and not in the landed commit..??

danbev commented:
I honestly did not notice the comments until after I had committed the
changes and pushed them. I did not intentionally ignore the comments and I
should have waited longer before merging. I was a little too eager  but
will make sure to allow more time for reviews. Sorry about this.

tisdag 6 september 2016 skrev Jeremiah Senkpiel <notifications@github.com>:

> Why was this landed without addressing the comments? I'm 99% certain I
> posted those here, and not in the landed commit..??
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/8405#issuecomment-244960847>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAaY3__HpJJhL4SuYUnTpFCkOL9BJvXAks5qnXNlgaJpZM4J0mlA>
> .
>
